erica/infrastructure/sqlalchemy/repositories/erica_request_repository.py

```diff
@@ -0,0 +1,20 @@
+from abc import ABC
+
+import injector as injector
+from sqlalchemy.orm import Session
+
+from erica.domain.EricaAuftrag.EricaAuftrag import EricaAuftrag
+from erica.domain.Repositories.EricaRequestRepositoryInterface import EricaRequestRepositoryInterface
+from erica.infrastructure.sqlalchemy.erica_request_schema import EricaRequestSchema
+from erica.infrastructure.sqlalchemy.repositories.base_repository import BaseRepository
+
+
+class EricaRequestRepository(
+    BaseRepository[EricaAuftrag, EricaRequestSchema],
+    EricaRequestRepositoryInterface,
+    ABC
+):
+    def __init__(self, db_connection: Session):
```

Comment:
You have removed the injection here - why?

Reply:
When we inject the db_connection in the InfrastructurModule we do not need it here. Furthermore, when we inject as a default value this leads to some problems:
- It's harder to mock it because, when we use the ericaRequestRepository in the tests without an opyoid configuration this will always set the connection to the one implied here. This lead to some errors in the code
- In python the default value can act as a singleton if it is an object. For the session that is fine but for other parts this can be problematic